srcs/web/app.py

```diff
@@ -16,6 +16,33 @@
 app.add_route('/metrics', prometheus_metrics)
 
 
+@app.on_event("startup")
+@repeat_every(seconds=60 * 60)
+def delete_orphan_runners():
+    metrics.runner_vm_orphan_delete.set(0)
+    metrics.runner_github_orphan_delete.set(0)
```

Comment:
If you set the metrics at 0 everytime we can this function, prometheus will not be able to track the amount of orphan you deleted.

Reply:
It will set the metrics every hour. 
Because if we don't reset it, it will always increment and never stop